packages/host/app/components/preview.gts

```diff
@@ -8,11 +15,42 @@ interface Signature {
   }
 }
 
+function a(thing: any){ return [thing] }
+
 export default class Preview extends Component<Signature> {
   <template>
-    <this.renderedCard />
+    <button {{on 'click' this.toggle}}>Toggle</button>
+    <AnimationContext @use={{this.transition}} >
+      {{#each (a this.renderedCard) as |Rc|}}
+        <div {{sprite id="first"}} >
+          <Rc />
+        </div>
+      {{/each}}
+    </AnimationContext>
   </template>
 
+  toggle = () => this.mode = !this.mode;
+  @tracked mode = true;
+
+  transition = (changeset: Changeset): AnimationDefinition => {
+    return {
+      timeline: {
+        type: 'parallel',
+        animations: [
+          {
+            sprites: changeset.keptSprites,
+            properties: {
+              size: {}
```

Comment:
Reminder that when rebasing, you'll have to change this to:
```
properties: {
  width: {},
  height: {},
}
```

Reply:
Thx 👍🏽